games-fps/eduke32/eduke32-20180508.6885.ebuild

```diff
@@ -0,0 +1,236 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eapi7-ver gnome2-utils toolchain-funcs xdg-utils
+
+MY_BUILD="$(ver_cut 2)"
+MY_DATE="$(ver_cut 1)"
+
+MY_PN_HRP="duke3d_hrp"
+MY_PN_OPL="duke3d_musopl"
+MY_PN_PSX="duke3d_psx"
+MY_PN_SC55="duke3d_music-sc55"
+MY_PN_XXX="duke3d_xxx"
+
+MY_PV_HRP="5.4"
+MY_PV_OPL="2.01"
+MY_PV_PSX="1.11"
+MY_PV_SC55="4.02"
+MY_PV_XXX="1.33"
+
+DESCRIPTION="An open source engine port of the classic PC first person shooter Duke Nukem 3D"
+HOMEPAGE="http://www.eduke32.com/"
+SRC_URI="http://dukeworld.com/eduke32/synthesis/${MY_DATE}-${MY_BUILD}/${PN}_src_${MY_DATE}-${MY_BUILD}.tar.xz
+	http://www.eduke32.com/images/eduke32_classic.png
+	hrp? ( http://www.duke4.org/files/nightfright/hrp/duke3d_hrp.zip -> ${MY_PN_HRP}-${MY_PV_HRP}.zip )
+	offensive? ( http://www.duke4.org/files/nightfright/related/${MY_PN_XXX}.zip -> ${MY_PN_XXX}-${MY_PV_XXX}.zip )
+	opl? ( http://www.moddb.com/downloads/mirror/95750/102/ce9e8f422c6cccdb297852426e96740a -> ${MY_PN_OPL}-${MY_PV_OPL}.zip )
+	psx? ( http://www.duke4.org/files/nightfright/related/duke3d_psx.zip -> ${MY_PN_PSX}-${MY_PV_PSX}.zip )
+	sc-55? ( http://www.duke4.org/files/nightfright/music/${MY_PN_SC55}.zip -> ${MY_PN_SC55}-${MY_PV_SC55}.zip )"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="BUILDLIC GPL-2 HRP"
+SLOT="0"
+IUSE="cdinstall demo flac gtk hrp offensive opengl opl png psx sc-55 sdl1 server sdk tools vorbis vpx xmp"
+REQUIRED_USE="cdinstall? ( !demo )
+	demo? ( !cdinstall )
+	hrp? ( ^^ ( demo cdinstall ) )
+	offensive? ( ^^ ( demo cdinstall ) )
+	opl? ( ^^ ( demo cdinstall )
+		!sc-55 )
+	psx? ( ^^ ( demo cdinstall ) )
+	sc-55? ( ^^ ( demo cdinstall )
+		!opl )
+	vpx? ( opengl )"
+
+S="${WORKDIR}/${PN}_${MY_DATE}-${MY_BUILD}"
+
+MY_DEPEND_RDEPEND="media-sound/timidity-freepats
+	sys-libs/zlib:=
+	flac? ( media-libs/flac )
+		gtk? ( x11-libs/gtk+:2 )
+	opengl? ( virtual/glu
+		virtual/opengl )
+	png? ( media-libs/libpng:0= )
+	sdl1? ( media-libs/libsdl[joystick,opengl?,sound,video,X]
```

Comment:
The SDL 1 build fails for me.
```
source/build/src/sdlayer.cpp: In function ‘void uninitsystem()’:
source/build/src/sdlayer.cpp:677:5: error: ‘SDL_GL_UnloadLibrary’ was not declared in this scope
```

Reply:
Hm, I tend to drop SDL1 completely, what do you think?

Reply:
After looking further, the SDL 1.2 source seems nowhere to define `SDL_GL_UnloadLibrary`, but 2.0 does. So I am dropping it.